typespec/Applications.Core/containers.tsp

```diff
@@ -125,9 +125,6 @@ model ContainerPortProperties {
   @doc("Protocol in use by the port")
   protocol?: PortProtocol;
 
-  @doc("Specifies a route provided by this port")
-  provides?: string;
```

Comment:
Do we need to do any updates to the samples repo? Like eshop and other apps?

Reply:
This has already been done 👍